### DISCUSSION TITLE
some functionality questions

### DISCUSSION THREAD

**Discussion by joebordes** [drakkan/sftpgo#422] [Q&A] [answered]
Hi, I dedicated some time to play with the software and have some doubts

- I can't access the web client, I get a "not found"
- I can't seem to connect to WebDAV using filestash (account not  valid)
- is there a way to have shared folders. So each user has their own private folder but then there are other groups of users that share folders together?

Thanks

**Comment by drakkan** [comment, marked as answer]
Hi,

to use the web client you need the current development version, you can use one of the edge variants if you are on Docker, or you can download a recent build artifact from [here](https://github.com/drakkan/sftpgo/actions/workflows/development.yml).

WebDAV is disabled by default, after enabling it within `sftpgo.json` please verify that it works using a client like nautilus, windows explorer, finder. Once you are sure it is correctly configured try filestash again, I never used it anyway if you provide the sftpgo logs and a network dump I'll try to give a look.

Shared folders are supported, please take a look [here](https://github.com/drakkan/sftpgo/blob/main/docs/virtual-folders.md), here is the relevant part

> The same virtual folder can be shared among users, different folder quota limits for each user are supported. Folder quota limits can also be included inside the user quota but in this case the folder is considered "private" and sharing it with other users will break user quota calculation. 

I plan to write a getting started guide to clarify the main concepts with usage examples (hopefully before v2.1) but I don't have a definite schedule for this, sorry, it depends on my free time. If you would like to submit a pull request to improve the docs it will be appreciated, thank you

**Comment by joebordes** [comment]
I got web client and directory sharing working (downloaded from github and built).

For webdav, I can access using Nautilus with no problem at all but Filestash says that there is an issue with the TLS certificates (which appear correctly in the browser)

`{"level":"error","time":"2021-05-17T10:50:17.087","sender":"webdavd","message":"http: TLS handshake error from xxx.xxx.xxx.xxx:50398: remote error: tls: bad certificate"}`

I suppose that is more a Filestash issue

Thanks for your help

**Comment by joebordes** [comment]
just tried deactivating TLS, normal http and it worked. So definitely something filestash is doing wrong. No worries

**Reply by drakkan** [reply]
Thank you for the update. As said above I have never used filestash, please let me know if you find anything interesting
